Automattic/wp-calypso
# Pro dashboard: Add official downtime monitoring upgrade banner.

Related to 1204992567518369-as-1204995191652299

## Proposed Changes

This PR adds the official Downtime monitoring upgrade banner. Note that the tracking of events and showing of the info modal will be addressed in a separate PR.

<img width="1416" alt="Screen Shot 2023-07-12 at 4 54 45 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/72048276-866f-4340-b9f5-0d5299631840">


## Testing Instructions

Since these changes are made specifically for agencies, you must set yourself(partner) as an agency - 2c49b-pb. Make sure to switch it back to the previous type.

* Use the Jetpack cloud live link and go to `/dashboard`.
* Confirm that the Downtime monitoring upgrade banner is visible.
* Confirm that clicking the explore or close button will dismiss the banner. The banner should no longer show up in the next page reload after it was dismissed.

**How to reset the banner?**

1. You will need to be on the staging site to access the reset functionality. (Go to https://cloud.jetpack.com/ and make sure you are running behind a proxy)
2. In the Jetpack page at the bottom right. Hover on the debug icon and click preferences. Look for **jetpack-dashboard-agency-program-downtime-monitoring-upgrade-banner-preference** and clear it by clicking the **X** button.
<img width="1174" alt="Screen Shot 2023-07-12 at 4 57 50 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/aeb0af42-c99b-409f-a652-fb2fb207b97b">
3. This should reset the banner and will show up again in the Dashboard screen.



## Pre-merge Checklist
